Transcelerator/TxlQuestions.xml

```diff
@@ -31822,11 +31880,17 @@
         <Questions scrref="JHN 12.33" startref="43012033" endref="43012033">
           <Q>What is the reason that Jesus said this?</Q>
           <A>He said it to show how he was to die. (33)</A>
+          <Alternative source="TomBogle">Why did Jesus say this?</Alternative>
+          <Alternative source="TomBogle">What was Jesus intending to communicate when he said this?</Alternative>
         </Questions>
         <Questions scrref="JHN 12.34" startref="43012034" endref="43012034">
           <Q>What is the reason that the people were confused?</Q>
           <A>They were confused because the law stated that the Messiah would live forever,</A>
           <A>so how could he say that "the Son of Man must die/be lifted up?" (34)</A>
+          <Alternative source="dryfgy">What is the reason that the people were confused about Jesus' words?</Alternative>
+          <Alternative source="TomBogle">Why the people were confused about Jesus' words?</Alternative>

```

Comment:
These three alternatives you added are not worded well. I think you intended "Why were..."

Reply:
Fixed